### DISCUSSION TITLE
flybywiresim / a32nx flight plan causes dropped frames

### DISCUSSION THREAD

**Discussion by ruimirada** [flybywiresim/aircraft#2714] [General]
I noticed after the last update 1.12.13.0
I noticed a big drop in frames and crashes during the flight.
I made a flight importing the simbrief plane and the frames dropped, then I made a flight plan manually and the frames dropped, and then I took off without a flight plan and I didn't have a frame drop, so I ask the question will be a flybywiresim problem / a32nx?
You would like someone to take the test and report your experience and openion here.

**Comment by Daboj1337** [comment]
Yes, there are some more posts about this issue here.

**Reply by ruimirada** [reply]
And does anyone do anything to solve this problem?

**Reply by Daboj1337** [reply]
No. But i found some other guys with the same problem on the msfs forum. Were not alone.

**Comment by andreda** [comment]
I am having crashes especially when I look down to turn on the WX radar. The entire computer crashes.... like totally freezes... and I have to hard shutdown. Is this what you are experiencing?

**Reply by ruimirada** [reply]
I didn't have this problem, so you can try to solve it, you must remove everything inside the Community folder, leaving only the A32NX to see if it happens again, if it doesn't happen again it is some folder that creates conflict, A32nx 0.51 is giving a lot of problems with the latest update.
